src/test/resources/pages/instructorFeedbackConstSumOptionQuestionAddSuccess.html

```diff
@@ -2124,7 +2124,9 @@ <h5 class="col-sm-2">
         </div>
         <div>
           <span class="pull-right">
-            <input class="btn btn-primary" id="button_submit_add" tabindex="9" type="submit" value="Save Question">
+              <button class="btn btn-primary" id="button_submit_add" tabindex="9" type="submit">
+                  Save Question
+              </button>
```

Comment:
I'm quite sure this is not regenerated with God mode. Can you regenerate the HTML files with God mode?